src/lib.rs

```diff
@@ -1227,15 +1228,14 @@ impl Function {
             _ => {
                 panic!("unhandled function type {}", function_type)
             }
-        };
-        f
+        }
     }
 }
 
 fn as_num(o: &Object) -> f64 {
     match *o {
         Object::Integer(i) => i as f64,
-        Object::Real(f) => f,
+        Object::Real(f) => f.into(),
```

Comment:
What's the build error being fixed here?

Reply:
f is of type f32, but signature wants f64, so this will convert into it